CMakeLists.txt

```diff
@@ -3,15 +3,38 @@ PROJECT(libgraphqlparser C CXX)
 
 SET(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake" "${CMAKE_MODULE_PATH}")
 
-INCLUDE(version)
+IF(UNIX)
+  SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++11")
+  SET(FLEX_COMPILE_FLAGS "--header-file=lexer.h")
+ELSEIF(WIN32)
+  # If we're building this with vcpkg on Windows, let portfile.cmake tell us where it
+  # stored these tools. Otherwise these variables should be empty and we'll fall back
+  # to the normal CMake FIND_PACKAGE logic for each of these programs.
+  SET(CMAKE_PROGRAM_PATH
+    "${VCPKG_DOWNLOADS_PYTHON2_DIR}"
+	"${VCPKG_DOWNLOADS_FLEX_DIR}"
+	"${VCPKG_DOWNLOADS_BISON_DIR}"
+	"${CMAKE_PROGRAM_PATH}")
+
+  SET(FLEX_COMPILE_FLAGS "--header-file=lexer.h --wincompat")
+
+  # Let CMake figure out the exports for the SHARED library (DLL) on Windows.
+  SET(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS TRUE)
+ENDIF()
 
-SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=gnu++11")
+INCLUDE(version)
 
 FIND_PACKAGE(PythonInterp 2 REQUIRED)
 IF (NOT PYTHON_VERSION_MAJOR EQUAL 2)
   MESSAGE(FATAL_ERROR "Python 2 is required.")
 ENDIF()
 
+IF(UNIX)
```

Comment:
FLEX_COMPILE_FLAGS was already set above. Please set it only once.

Reply:
Yup, looks like I accidentally duplicated it.